### PR TITLE
PullRequest_firstReview

### PR DESCRIPTION
**Pull request by SebastianSolisFenger**
This is the pull request for the review of my projects from the microverse trials.
 Learning objectives:

- Apply best practices in HTML code. 
- Use semantic HTML tags. 
- Use CSS selectors correctly. 
- Use CSS box model. 
- Perform a code review on a past project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+        <div class="subscription">
+
+            <h2><b><strong>Home-made masterpieces</strong></b></h2>
+
+            <form id="form" action="https://codepen.io/freeCodeCamp/full/RKRbwL">
+                <input id="email" type="email" placeholder="your email...">
+                <input id="input" type="submit" value="Let's Go!">
+            </form>
+        </div>
```
Comment (SebastianSolisFenger) [inline index.html]: Please, kindly change the div tag of each section of your code for the sections tag.
For example:   <section class="subscription">  </section class="subscription">

Please, make sure to Include: subscription, featuresSec, videoSec and pricing Sec.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+
+#header {
+  padding: 24px 25px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: rgb(143, 142, 241);
+  border-radius: 10px;
+  position: relative;
+  width: 95%;
```
Comment (SebastianSolisFenger) [inline style.css:26]: Please, kindly change the width from 95% to 100% so it matches with the rest of the sections.

**Review by SebastianSolisFenger — COMMENTED** [review]
Hi @SebastianSolisFenger! Good job so far! 



# changes required ♻️ 

- Please check all my comments below and make all required changes
- Add a README.md file providing some information about your project.
- Check linter errors (Webhint and Stylelint).

Submit for a re-review when you are done.

Thanks!``

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @SebastianSolisFenger, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**